<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_editor</class>
 <widget class="QWidget" name="data_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="data_text_edit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
